lib/apiservers/engine/backends/container.go

```diff
@@ -1349,6 +1362,38 @@ func setPathFromImageConfig(config, imageConfig *containertypes.Config) {
 func validateCreateConfig(config *types.ContainerCreateConfig) error {
 	defer trace.End(trace.Begin("Container.validateCreateConfig"))
 
+	// process cpucount here
+	var cpuCount int64 = DefaultCPUs
```

Comment:
Just saying: another way of defining variable of a specific type is cpuCount := int64(DefaultCPUs)